### PR TITLE
API 10293 Populate New Immunization Fields

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java [hunk 1]
```diff
@@ -311,6 +311,18 @@
             .build();
       };
 
+  private Function<DatamartImmunization, ImmunizationEntity> toImmunizationEntity =
```
Comment (jblefkove-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:314]: alphabetize?

Comment (that-guy-is-it) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:314]: Should I move `toDiagnosticReportEntity` too? It also appears out of order. Boy Scout Principle and all that.

Comment (jblefkove-va) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:314]: yep

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**